common/sgx/verifier.c

```diff
@@ -456,8 +456,9 @@ oe_result_t oe_sgx_extract_claims(
 
     // Note: some callers can have custom_claims_buffer pointing to a non-NULL
     // buffer containing a zero-sized array.
-    if (!format_id || !report_body || !report_body_size ||
-        (!custom_claims_buffer && custom_claims_buffer_size))
+    if (!format_id || !report_body || !report_body_size || !claims_out ||
+        !claims_length_out ||
+        (!custom_claims_buffer != !custom_claims_buffer_size))
```

Comment:
This PR doesn't touch include\openenclave\internal\sgx\plugin.h but that's where the doxygen for oe_sgx_extract_claims() is and `custom_claims_buffer` and size are not stated as optional there, so I think we need to update that file too (I will also observe that the `sgx_endorsements` param is out of order in the param list in the doxygen so if you update the file maybe fix that too)

Reply:
Good catch! I'll fix. On the other hand, `include\openenclave\internal\sgx\plugin.h` is internal, so OE SDK application developers don't see this header file or its doxygen document.